go.mod

```diff
@@ -34,17 +34,17 @@ require (
 	k8s.io/klog/v2 v2.110.1
 	k8s.io/kube-aggregator v0.29.0
 	k8s.io/utils v0.0.0-20240102154912-e7106e64919e
-	open-cluster-management.io/addon-framework v0.8.1-0.20240205013730-13fbb6259464
+	open-cluster-management.io/addon-framework v0.9.1-0.20240311065811-974d3377ecbd
 	open-cluster-management.io/api v0.13.0
 	open-cluster-management.io/sdk-go v0.13.1-0.20240306030534-3142983462d9
-	sigs.k8s.io/controller-runtime v0.16.2
+	sigs.k8s.io/controller-runtime v0.17.0
```

Comment:
Please use `sigs.k8s.io/controller-runtime v0.17.2` . The Restmapper is buggy before that.